plugins/package.json

```diff
@@ -25,6 +26,15 @@
     ]
   },
   "devDependencies": {
-    "cordova-ios": "^6.2.0"
+    "@babel/preset-typescript": "^7.18.6",
+    "babel": "^6.23.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.5",
+    "cordova-ios": "^6.2.0",
+    "source-map-loader": "^4.0.0",
```

Comment:
are all of the dependencies here used ? For example, `source-map-loader` or `babel-loader` are not in webpack config. Or they are required by other dependencies? Could you describe what are they for and how applied

Reply:
these dependencies were auto generated in one of the iterations of trying to make the bundling work, I will delete them